editor/sidebar/post-visibility/index.js

```diff
@@ -71,6 +74,21 @@ class PostVisibility extends Component {
 		this.setState( { opened: false } );
 	}
 
+	handleBlur( event ) {
+		if ( this.state.opened && event.relatedTarget !== null && ! this.dialog.contains( event.relatedTarget ) ) {
```

Comment:
While explicitness could be handy, I don't know that there's ever a case that `event.relatedTarget` is `null` and `this.dialog.contains( null )` would ever be true. In other words, we could safely drop `&& event.relatedTarget !== null`.